content/learn/book/storing-data/queries.md

```diff
@@ -6,10 +6,352 @@ weight = 4
+    // The `mut life` tells Rust that we want to mutate the Rust variable
+    // and `.iter_mut` tells Bevy that we want to access the query data mutably,
+    // rather than downgrading it to a read-only `&Life`
+    for mut life in poisoned.iter_mut(){
+        // Simply calling life.value -= 1 will underflow!
+        life.value.saturating_sub(1);
+    }
+}
+```
+
+{% callout(type="info") %}
+
+You can include multiple queries within a single system, allowing you to access component data in more flexible ways.
+But, if Bevy is handing out mutable references to component data in safe Rust, how does it ensure that users don't
+invoke undefined behavior due to the forbidden [mutable aliasing]?
+
+Bevy protects against this by examining the [`Access`] of each of the system params in each systems,
+then panicking if they could conflict.
+
+System params conflict if the data  they are accessing overlaps and at least one of the accesses are mutable.
+You can avoid this by ensuring that your access is provably disjoint: `Without` can be very helpful.
+
+If you run into this, you'll be pointed to the [B0002] error page,
+which has advice on how to fix and avoid this problem.
+
+{% end %}
+
+By changing our [`QueryData`] terms from `&Life` to `&mut Life`, we change the type of [query item] returned,
+changing the type of object we get when we iterate over our queries.
+`Query<&Life>` corresponds to a `&Life`, giving us direct read-only access to data inside of our world.
+However, you may have noticed that `Query<&mut Life>` returns a [`Mut<Life>`]. Why?
+
+This [smart pointer] wraps a `&mut T` and allows Bevy to automatically detect changes.
+While this is talked about in more depth in the chapter on [change detection],
+it's helpful to know that [`Changed`] and [`Added`] are both query filters.
+
+[reference]: https://doc.rust-lang.org/book/ch04-02-references-and-borrowing.html
+[query item]: https://dev-docs.bevy.org/bevy/ecs/query/trait.QueryData.html#associatedtype.Item
+[`Mut<Life>`]: https://dev-docs.bevy.org/bevy/ecs/change_detection/struct.Mut.html
+[smart pointer]: https://doc.rust-lang.org/book/ch15-00-smart-pointers.html
+[change detection]: ../control-flow/change-detection.md
+[mutable aliasing]: https://doc.rust-lang.org/rust-by-example/scope/borrow/alias.html
+[`Access`]: https://dev-docs.bevy.org/bevy/ecs/query/struct.Access.html
+[B0002]: https://bevy.org/learn/errors/b0002/
+
+## Accessing data on specific entities
+
+While many systems will operate by simply iterating over all of the entities in a query,
+it is often helpful to look up (and possibly mutate) the data of a specific entity.
+
+This is fast and easy to do, using [`Query::get`] and its mutable sibling [`Query::get_mut`].
+These respect the query data and query filters of the query they are called on,
+making them an extremely powerful tool.
+
+Of course, this begs the question: where do we get the [`Entity`] identifier.
```

Comment:
Pedantically speaking, this is not what the term "begging the question" means: https://en.wikipedia.org/wiki/Begging_the_question - however, this misuse of the term is so common in popular writing that I'm tempted to let it pass.

Still, something like "raises the question" would be more correct.

Reply:
```suggestion
Of course, this raises the question: where do we get the [`Entity`] identifier.
```